airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py

```diff
@@ -53,6 +53,29 @@ def connector_spec_dict_fixture(actual_connector_spec):
     return json.loads(actual_connector_spec.json())
 
 
+@pytest.fixture(name="secret_property_names")
+def secret_property_names_fixture():
+    yield (
```

Comment:
nit: any reason to use yield instead of return here?

Reply:
that's a habit based on https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#yield-fixtures-recommended
specifically in this case gives no profit, changed to `return`